pkg/ssh/connection_golang.go

```diff
@@ -209,7 +210,7 @@ func GetUserInfo(uri *url.URL) (*url.Userinfo, error) {
 // ValidateAndConfigure will take a ssh url and an identity key (rsa and the like) and ensure the information given is valid
 // iden iden can be blank to mean no identity key
 // once the function validates the information it creates and returns an ssh.ClientConfig.
-func ValidateAndConfigure(uri *url.URL, iden string) (*ssh.ClientConfig, error) {
+func ValidateAndConfigure(uri *url.URL, iden string, insecureMachine bool) (*ssh.ClientConfig, error) {
```

Comment:
Non-blocking: Naming this same as the `…Options` field would be nice. It’s used infrequently enough that a long descriptive name seems worth it to me.